deployment/deploymentConfigurations.json

```diff
@@ -34,13 +34,43 @@
         }
       }
     },
-    "aave-v2": {
+    "aave-v2-forks": {
       "aave-v2": {
+        "mainnet-arc": {
+          "template": "aave.v2.template.yaml",
+          "messari": "messari/aave-v2-ethereum-arc",
```

Comment:
I think they should be more appropriately named `aave-v2-arc-ethereum` as technically `aave-v2-arc` can be deployed on different chains

Reply:
Alternatively we can just have `aave-arc-ethereum` and `aave-rwa-ethereum` as that seems to align with how they market themselves (no v2)